practice_app/pages/api/getStatistics.test.js

```diff
@@ -0,0 +1,23 @@
+const { checking } = require('./getStatisticsModule')
+
+describe('getStatistics tests', () => {
```

Comment:
describe string should contain the exact name of the function you are testing, in my opinion. So it should `describe('checking')`

Reply:
done